lib/Ix/Context/WithAccount.pm

```diff
@@ -61,6 +61,16 @@ sub _build_state ($self) {
   });
 }
 
+has scratchpad => (
```

Comment:
We should have some comments explaining what the scratchpad's scope is.

It looks like it's limited to a single JMAP Method and any JMAP calls made internally by that method, but not other Methods after the main method has finished executing